pkg/kubelet/kubelet_node_status.go

```diff
@@ -467,6 +467,10 @@ func (kl *Kubelet) tryUpdateNodeStatus(tryNumber int) error {
 		}
 	}
 
+	if err := kl.containerManager.UpdateNodeAllocatableCgroups(node.Status.Capacity); err != nil {
+		return err
```

Comment:
is there a benefit to exiting on the error here versus logging the error?  i would want to be able to update node status even if this failed.

Reply:
i think i would prefer if we log the error (although if it errors an event is already going to be thrown in the enforce call).

can we remove the `return err` and log instead?

Reply:
Done